src/textual/message_pump.py

```diff
@@ -311,6 +311,20 @@ async def emit(self, message: Message) -> bool:
         else:
             return False
 
+    async def dispatch_key(self, event: events.Key) -> None:
```

Comment:
Should a "message pump" know anything about key events? Could this method live elsewhere in the hierarchy?

Reply:
The only only other option is DOM Node, but I'd like to keep that free of event handlers.

You're right though, it is a little incongruent. I'll leave a comment to think about that one later.